concrete/attributes/user_selector/controller.php

```diff
@@ -37,8 +37,9 @@ public function getDisplayValue()
     {
         $uID = $this->getAttributeValue()->getValue();
         $user = User::getByUserID($uID);
+        $ui = $user->getUserInfoObject();
         if (is_object($user)) {
-            return t('<a href="%s">%s</a>', $user->getUserName(), $user->getUserPublicProfileUrl());
+            return t('<a href="%s">%s</a>', $user->getUserName(), $ui->getUserPublicProfileUrl());
```

Comment:
How can this ever generate a correct link? In this case the user name is the first parameter (%s). Or am I missing something?